jobs/async-upload/job/models.py

```diff
@@ -158,6 +158,61 @@ def validate_registry_config(self) -> 'RegistryConfig':
         return self
 
 
+class RegisteredModelMetadata(BaseModel):
+    """Metadata for creating a RegisteredModel."""
+    name: str | None = None
+    id: str | None = None  # Alternative to name for existing models
```

Comment:
I'm not sure I understood the meaning of this comment, please? 🤔

Reply:
on second pass, got it:

```suggestion
    id: str | None = None  # valorized when RegisteredModel exists on server
```

wdyt?